src/app/landing/LandingPage.css

```diff
@@ -248,4 +248,12 @@
   .LandingPage h3 {
     font-size: 11px;
   }
+
+  .LandingPage .IntroBody {
+    background: linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.6)),url('/img/landing_page/top1_mobile.png') no-repeat center center fixed;
+    background-position: 30% 0%;
+    background-size: cover;
+  /* color: #FFFFFF; */
```

Comment:
Why the duplication?

Reply:
I'm experiencing a CSS bug.
Without duplicating the background-position and background-size, I get this:
![image](https://user-images.githubusercontent.com/20152824/82692885-80b51080-9c2e-11ea-9a70-758e0582f74b.png)

I think setting background on this more specific selector overrides/ resets background-position and background-size.
Also removed the commented out CSS

Reply:
Hmm, but the selector looks exactly the same: it's `.LandingPage .IntroBody` at the top of the file too.
It's probably something small we're missing... But the duplication, with the exact same selector, can't possibly be necessary.